site/src/pages/AIBridgePage/RequestLogsPage/filter/filter.tsx

```diff
@@ -12,10 +13,16 @@ const AIBRIDGE_PROVIDERS: SelectFilterOption[] = [
 	{
 		label: "OpenAI",
 		value: "openai",
+		startIcon: (
+			<ExternalImage src="/icon/openai.svg" className="size-icon-sm" />
```

Comment:
the first mapping was this here

it might not actually be important to consolidate, it just caught my eye